src/Pond.tsx

```diff
@@ -46,10 +72,35 @@ let singletonPond: PondType | undefined = undefined
 /**
  * Top level component to initialize the pond
  *
- * ## Example:
+ * ## Minimal example:
+ * ```js
+ * ReactDOM.render(
+ *   <Pond onError={onError}>
+ *     <App />
+ *   </Pond>,
+ *   document.getElementById('root')
+ * )
+ * ```
+ * 
+ * ## Complete example:
  * ```js
  * ReactDOM.render(
- *   <Pond>
+ *   <Pond onError={onError}
+ *    manifest={{
+ *     appId: 'io.actyx.react-pond-example',
+ *     displayName: 'React Pond Example',
+ *     version: '0.0.1',
+ *     signature: 'v2tz...JBPT3/'
+ *   }}
+ *   connectionOpts={{
+ *     actyxHost: 'actyx',
```

Comment:
should this not be localhost
```suggestion
 *     actyxHost: 'localhost',
```

Reply:
Hm, `localhost` is the default, so it wouldn't make much sense as an example.

Reply:
It looks like, you want to show the options. 

Using another host in the react pond is the worth we like to see :-)

Reply:
I don't understand that sentence, sry.